ui/src/app/views/concept-homepage/component.html

```diff
@@ -65,13 +75,14 @@
                          [placeholderValue]="placeholderValue"
                          (getSelectedConcepts)="selectConcept($event)">
       </app-concept-table>
-      <div *ngIf="conceptSelected" (mouseenter)="addTextHovering=true" (mouseleave)="addTextHovering=false" (click)="openAddModal()" class="add-section">
-        <div class="add-text" [class.hovering]="addTextHovering">Add to set</div>
-        <app-sliding-fab [expanded]="'Add to set'" (submit)="openAddModal()">
+      <div (mouseenter)="addTextHovering=true" (mouseleave)="addTextHovering=false" (click)="openAddModal()">
+        <div [class.hovering]="addTextHovering">Add to set</div>
+        <app-sliding-fab [expanded]=addToSetText (submit)="openAddModal()" [disable]="!isConceptSelected">
           <clr-icon class="add-icon" shape="plus"></clr-icon>
         </app-sliding-fab>
       </div>
     </div>
   </app-top-box>
 </ng-template>
-<app-concept-add-modal [selectedDomain]="selectedDomain.domain" [selectedConcepts]="selectedConcept"></app-concept-add-modal>
+<app-concept-add-modal [selectedDomain]="selectedDomain.domain" [selectedConcepts]="selectedConcept"
```

Comment:
So not sure if this is a real issue, but I am seeing an error in my console for this, I think because it's not actually set yet. (Or rather the value it's set to isn't set yet?)

Reply:
Yes, this has been addressed by intializing selectedDomain once we get the result from  this.conceptsService.getDomainInfo